webpack.config.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
   output: {
     path: path.resolve(__dirname, "dist"),
     filename: "[name].bundle.js",
-    publicPath: path.resolve("/"),
+    publicPath: "/",
```

Comment:
I'm not certain if this change will work on all platforms? Consider windows where `/` is a valid path separator (cf. UTF-16). I think this is also a relative path appended to CWD of the node.js process? Or does `webpack` handle this properly?

Reply:
I hope it's the latter. You see the reason I change it to `"/"` only because when I tried to build the app on a windows machine, all the resource file in the index.html are pointed to the directory `"D:\/"`. But with `"/"`, it works on both windows and mac. So I guess webpack did do something here.

Reply:
I believe then `webpack` canonicalize paths probably.